sigridci/sigridci.py

```diff
@@ -127,10 +127,8 @@ def submitUpload(self, options, systemExists):
         analysisId = uploadLocation["ciRunId"]
         log(f"Sigrid CI analysis ID: {analysisId}")
         log("Publishing upload" if self.publish else "Submitting upload")
-
-        if not self.uploadBinaryFile(uploadUrl, upload):
-            raise Exception("Uploading file failed")
-            
+        self.uploadBinaryFile(uploadUrl, upload)
+        
```

Comment:
Do we want to add an "Upload successful" (or similar) log message for completeness?